internal/expr/typeinfo.go

```diff
@@ -53,8 +53,15 @@ func (f structField) memberName() string {
 	return f.tag
 }
 
+// typeInfo exposes useful information about types used in SQLair queries.
 type typeInfo interface {
 	typ() reflect.Type
+
+	// typeMember returns the type member associated with a given column name.
+	typeMember(string) (typeMember, error)
```

Comment:
Having the parameter name here makes the code readable without having to read the docs above, so it's typically a good convention even if it's not strictly required.

Reply:
Good point, changed.